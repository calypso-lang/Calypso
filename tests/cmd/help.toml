bin.name = "calypso"
args = "--help"
