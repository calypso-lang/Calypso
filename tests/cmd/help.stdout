calypso 

USAGE:
    calypso [OPTIONS] <SUBCOMMAND>

OPTIONS:
        --color <COLOR>
            Set how color is displayed, if at all. By default this is set to `auto`.
            
            Possible values:
            
            - `always`: Always use color, even if stdout/stderr is not a TTY
            
            - `ansi`: Always use color, using ANSI escape codes, even if stdout/stderr is not a TTY
            or does not support them.
            
            - `auto`: Use color if stdout/stderr is a TTY, don't if it is not.
            
            - `never`: Never use color, even if stdout/stderr is a TTY.
            
            [default: auto]
            [possible values: always, ansi, auto, never]

    -h, --help
            Print help information

        --log <LOG>
            The logging filter to use.
            
            See tracing-subscriber's EnvFilter type for an explanation of the format:
            
            https://docs.rs/tracing-subscriber/*/tracing_subscriber/filter/struct.EnvFilter.html
            
            [env: CALYPSO_LOG=]

        --log-format <LOG_FORMAT>
            The logging format to use.
            
            Formats available:
            
            - `pretty`: Overly verbose, but good for human consumption.
            
            - `compact`: Less verbose, but still readable.
            
            - `json`: Intended for machine interpretation. (see
            https://docs.rs/tracing-subscriber/*/tracing_subscriber/fmt/format/struct.Json.html)
            
            [env: CALYPSO_LOG_FORMAT=]
            [default: compact]
            [possible values: pretty, compact, json]

SUBCOMMANDS:
    explain
            Explain an error that has detailed information on troubleshooting [aliases: expl, exp,
            ex]
    help
            Print this message or the help of the given subcommand(s)
    internal
            Commands used for debugging Calypso's internals and implementation [aliases: int]
